api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
To be honest, you guys know better how you want to sort orders within a group, i for one have to word on this, my assumption is that orders with no sequence number get pushed to the end of the set just like null orders if any